### ISSUE TITLE
Keras Convolutional Layer Bias Weight using Tensorflow backend

### ISSUE THREAD

**Issue by ghostplant** [keras-team/keras#10354]
I have a question about the Keras CNN Layer process on Tensorflow. According to Keras summary, the CNN layer is expected to include not only the kernel weights (e.g. 3 x 3) but also the bias weights and thus makes it 9 + 1 = 10 in total for a single convolutional kernel.

However, I profiled the Tensorflow-backend syscall and I see not bias weight is managed by Tensorflow, so does it mean Tensorflow backend for Keras doesn't support handling CNN bias weight?

**Comment by Dref360** [comment]
The bias is not handled by the backends, it's defined in the Keras layer and added to the output.
https://github.com/keras-team/keras/blob/master/keras/layers/convolutional.py#L139

**Comment by ghostplant** [comment]
@Dref360 Thanks, so where is the gradient calculation of Conv Bias when doing backward training?

**Comment by Dref360** [comment]
What do you mean? The gradient is computed by AD (from K.gradients) just like every other ops.

**Comment by ghostplant** [comment]
Thanks, got it. So the gradient calculation is automatically generated by backends, even including the bias_add.
